email/mjml/common/footer.mjml

```diff
@@ -8,8 +8,8 @@
 					padding="0px 0px 6px"
 				/>
 				<mj-text font-size="12px" line-height="18px">
-					​​You are receiving this email from the VIP Security Boost to ensure
-					you are aware of any security-related change in your account.
+					​​You are receiving this email from the WordPress VIP WordPress
+					Security Control feature to keep you informed of this security change.
 				</mj-text>
```

Comment:
Do we need the full "WordPress VIP WordPress Security Control" name here? Maybe we can skip the "WordPress VIP" prefix here since it's already on the email subject and we also have a VIP logo on the email.